.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.17.5
```

Comment:
Actually looks like we can specify `lts` as a value.  Lets try that.

![image](https://user-images.githubusercontent.com/3020266/153249122-765f1d45-8333-4253-b28d-fc1c920cd3d3.png)

Reply:
@mathewjordan Yes, let's keep `lts` as a value and keep it in the repo, committed.